<?xml version="1.0"?>
<launch>
    <node pkg="Barcode_Cam" type="Barcode_Reader" name="Barcode_Reader" output="screen">
        <param name="ip" value="192.125.1.20"/>
        <param name="port" value="3000"/>
        <param name="spin_rate" value="30"/>
        <param name="len_name" value="8"/>

    </node>
</launch>
